src/page-components/projects/sections/highlighted-project/styles.ts

```diff
@@ -0,0 +1,79 @@
+import styled from 'styled-components'
```

Comment:
@HormCodes co muzeme udelat pro mobilni versie?

Reply:
Omlouvám se, měl jsem za to, že je tam jen jeden design, ale jsou tam dva. Zeptal jsem se Alberta a čekám na odpověď.

Reply:
Tak víceméně jít logickou cestou a zkombinovat v3 a v1. Tedy konkrétně highlighted projekt jak je ve v3 (viz screenshot) a poté karty projektů pod sebe. 
<img width="167" alt="Screenshot 2021-02-09 at 09 29 03" src="https://user-images.githubusercontent.com/10332399/107338343-b7cc9a00-6abb-11eb-9475-a181ed80aa7e.png">